code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -396,6 +396,26 @@
 	list_reagents = list("grenadine" = 100)
 	foodtype = FRUIT
 
+/obj/item/reagent_containers/food/drinks/bottle/applejack
+	name = "Buckin' Bronco's Applejack"
+	desc = "Kicks like a horse, tastes like an apple!"
+	custom_price = 10
+	icon_state = "applejack_bottle"
+	isGlass = TRUE
+	list_reagents = list("applejack" = 100)
```

Comment:
I would have thought this would be a reaction involving apple juice

Reply:
ok, just as long as it's clear that as-is this reagent is pretty limited rn.

Reply:
I could add in another cocktail if you'd like for it to have more to interact with

Reply:
when I say limited I mean it's only found in a bottle in a vending machine, not in a dispenser or anything else.